CONTRIBUTING.md

```diff
@@ -180,4 +180,21 @@ make misspell-correction
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
+  - Do not amend ECS names to follow plural guidelines or other syntactical
+    attribute naming rules.
```

Comment:
It's a bit unclear what are the "plural guidelines or other syntactical attribute naming rules".

Also, is this file the best place to define it? Maybe it should go to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md where all the guidelines are defined? It seems like we are scattering the guidelines across different places

Reply:
The linked file is metrics specific though, no?
I had found https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md#name-pluralization-guidelines about the pluralization rule which seems to be more generic?

Maybe https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/README.md is a more generic one for this repo. I'm not sure if it's searchable in the docs page though.

For now I would be fine with keeping it in the contribution guide as well.

Reply:
@dmitryax does that resolve your comment?

Reply:
@dmitryax can we resolve this?